packages/webapp/src/server.js

```diff
@@ -24,7 +25,6 @@ const createServer = async options => {
             abortEarly: false
           }
         },
-        cors: true,
         security: true
```

Comment:
I think security:true can be removed as well because the custom response headers replace what cors:true and security:true provide.

Reply:
ok, done